spec/controllers/master_files_controller_spec.rb

```diff
@@ -228,6 +228,23 @@ class << file
       expect(controller.current_ability.can?(:destroy, master_file)).to be_falsey
       expect(post(:destroy, params: { id: master_file.id })).to render_template('errors/restricted_pid')
     end
+
+    context "master file with child supplemental files" do
+      let!(:master_file) { FactoryBot.create(:master_file, :with_media_object, :cancelled_processing, supplemental_files: [supplemental_file]) }
+      let(:supplemental_file) { FactoryBot.create(:supplemental_file) }
+
+      around(:example) do |example|
+        # In Rails 5.1+ this can be restricted to whitelist jobs allowed to be performed
```

Comment:
I guessing this was copied from other examples in the test suite, but I think the comment can be dropped as it isn't adding much (and removes the problematic "whitelist").
```suggestion
```